### PR TITLE
[DOC][1/3] Documentation for builtin Outlier function

### PR DESCRIPTION
**Pull request by wizard-420**
How to initialize a boolean variable in dml?

### CODE REVIEW

File: dev/docs/builtins-reference.md [hunk 1]
```diff
@@ -322,3 +322,11 @@ y = X %*% rand(rows=ncol(X), 1)
 w = lm(X = X, y = y)
 ress = slicefinder(X = X,W = w, Y = y,  k = 5, paq = 1, S = 2);
 ```
+##outlier-Function
```
Comment (j143) [inline dev/docs/builtins-reference.md:325]: Thanks a lot, @wizard-420 ! Try to add usage and example and follow the same pattern as used for other functions. :)

Comment (wizard-420) [reply dev/docs/builtins-reference.md:325]: I need help with how to initialize a boolean data type.

**Comment by j143** [comment]
> How to initialize a boolean variable in dml?
For a variable add `0` or `1`, that would automatically allocates `double` or `boolean`.

For example:
```
x = 0;
```
Now, `x` can be considered `false`.
DML figures out `x` should used as a boolean or not based on usage of the`x` in the program.

**Comment by j143** [comment]
Hi @wizard-420 - please move the discussion to this place regarding `outlier` function, this way we are more likely receive other contributors help.

File: dev/docs/builtins-reference.md (hunk 1 above)
Comment (wizard-420) [reply dev/docs/builtins-reference.md:325]: I have added the contents. please check if everything is alright

**Comment by j143** [comment]
Now, you might want to work on `outlierByIQR` https://github.com/apache/systemml/blob/faec15810e18fc497e950b830811a919f5aa5c5f/scripts/builtin/outlierByIQR.dml#L23

in a different PR. I will review this and merge this later today.

**Comment by wizard-420** [comment]
ok I will get on it asap

**Comment by j143** [comment]
Thank you, @wizard-420 - LGTM. 👍 
🎉 🚀

**Pull request closed without merging**